src/Assets/Data.jsx

```diff
@@ -0,0 +1,151 @@
+import { v4 as uuid } from "uuid";
+
+
+export const products = [
+  {
+    _id: uuid(),
```

Comment:
i suggest don't use uuid for id instead use any random combination of letters and numbers. because for now it is fine but when u work on single product page, it will show error after refreshing bcz uuid changes after every render.

Reply:
Makes sense, will change it.